docker-compose.yml

```diff
@@ -149,8 +149,7 @@ services:
     volumes:
       - vault-config:/vault/config:z
       - consul-scripts:/consul/scripts:ro,z
-      - /tmp/edgex/secrets/ca:/tmp/edgex/secrets/ca:ro,z
-      - /tmp/edgex/secrets/edgex-kong:/tmp/edgex/secrets/edgex-kong:ro,z
+      - /tmp/edgex/secrets:/tmp/edgex/secrets:z
```

Comment:
Should it be `ro, z` for R/W?

Reply:
Default is read/write.  "z" flag for shared access.  So just "z" is shared read/write.

Reply:
Reason for the change.  Before, security-secrets-setup wrote PKI to /tmp/edgex/secrets.  In this next release, security-secretstore-setup will also write per-service Vault tokens under here as well.